app/controllers/api/providers_controller.rb

```diff
@@ -85,6 +85,21 @@ def import_vm_resource(type, id = nil, data = {})
       end
     end
 
+    def options
+      if params[:ems_type].blank?
+        all_options = Hash[
+          ExtManagementSystem.descendants.collect do |ems|
+            [ems.ems_type, ems.respond_to?(:provider_settings) ? ems.provider_settings : {}]
```

Comment:
Right. But here we are not serving the values of those options but the answer to the question "what per-provider options are for this provider type?". This is the static description of those options. please look at https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/45/files#diff-47711b761083b6ce40dacb7d3ee68f8b for example